services/graph/pkg/service/v0/driveitems.go

```diff
@@ -234,6 +245,231 @@ func (g Graph) GetDriveItemChildren(w http.ResponseWriter, r *http.Request) {
 	render.JSON(w, r, &ListResponse{Value: files})
 }
 
+// Invite invites a user to a storage drive (space).
+// fixMe:
+// - check permissions if user is capable of inviting
+func (g Graph) Invite(w http.ResponseWriter, r *http.Request) {
+	gatewayClient, err := g.gatewaySelector.Next()
+	if err != nil {
+		g.logger.Debug().Err(err).Msg("selecting gatewaySelector failed")
+		errorcode.GeneralException.Render(w, r, http.StatusInternalServerError, http.StatusText(http.StatusInternalServerError))
+		return
+	}
+
+	driveID, err := storagespace.ParseID(chi.URLParam(r, "driveID"))
+	if err != nil {
+		g.logger.Debug().Err(err).Msg("could not parse driveID")
+		errorcode.InvalidRequest.Render(w, r, http.StatusBadRequest, "invalid driveID")
+		return
+	}
+
+	itemID, err := storagespace.ParseID(chi.URLParam(r, "itemID"))
+	if err != nil {
+		g.logger.Debug().Err(err).Msg("could not parse itemID")
+		errorcode.InvalidRequest.Render(w, r, http.StatusBadRequest, "invalid itemID")
+		return
+	}
+
+	if driveID.GetStorageId() != itemID.GetStorageId() || driveID.GetSpaceId() != itemID.GetSpaceId() {
+		g.logger.Debug().Interface("driveID", driveID).Interface("itemID", itemID).Msg("driveID and itemID do not match")
+		errorcode.InvalidRequest.Render(w, r, http.StatusBadRequest, "driveID and itemID do not match")
+		return
+	}
```

Comment:
Can we put this into a separate method/function I have the feeling we need this for all the other `/drives/{drive-id}/items/{item-id}/` related requests as well?